hmy/api_backend.go

```diff
@@ -127,53 +130,63 @@ func (b *APIBackend) ProtocolVersion() int {
 // Filter related APIs
 
 // GetLogs ...
+// TODO: this is not implemented or verified yet for harmony.
 func (b *APIBackend) GetLogs(ctx context.Context, blockHash common.Hash) ([][]*types.Log, error) {
 	// TODO(ricl): implement
 	return nil, nil
 }
 
 // HeaderByHash ...
+// TODO: this is not implemented or verified yet for harmony.
 func (b *APIBackend) HeaderByHash(ctx context.Context, blockHash common.Hash) (*types.Header, error) {
 	// TODO(ricl): implement
 	return nil, nil
 }
 
 // ServiceFilter ...
+// TODO: this is not implemented or verified yet for harmony.
 func (b *APIBackend) ServiceFilter(ctx context.Context, session *bloombits.MatcherSession) {
 	// TODO(ricl): implement
 }
 
-// SubscribeNewTxsEvent ...
+// SubscribeNewTxsEvent subcribes new tx event.
+// TODO: this is not implemented or verified yet for harmony.
 func (b *APIBackend) SubscribeNewTxsEvent(ch chan<- core.NewTxsEvent) event.Subscription {
 	return b.hmy.TxPool().SubscribeNewTxsEvent(ch)
 }
 
-// SubscribeChainEvent ...
+// SubscribeChainEvent subcribes chain event.
+// TODO: this is not implemented or verified yet for harmony.
 func (b *APIBackend) SubscribeChainEvent(ch chan<- core.ChainEvent) event.Subscription {
 	return b.hmy.BlockChain().SubscribeChainEvent(ch)
 }
 
-// SubscribeChainHeadEvent ...
+// SubscribeChainHeadEvent subcribes chain head event.
+// TODO: this is not implemented or verified yet for harmony.
```

Comment:
I think some of these are already working now, as I can listen to filterChange of newBlockFilter